core/rewards.lua

```diff
@@ -29,7 +30,9 @@ function Reward:Initialize(attrs)
 end
 
 function Reward:IsEnabled()
-    if self.class and self.class ~= ns.class then return false end
+    if self.class and (string.match(self.class, ns.class) == nil) then
+        return false
+    end
```

Comment:
Does `~=` not work here as expected? If there is a bug, wouldn't we need to fix the identical `ns.faction` line below?

Reply:
I see in 252159c42d76e6c41f2b30e9d1a8c83b54c7625c it is intended to support a space-delimited list of class names for rewards, but we do not utilize this functionality anywhere I can see in the current code after all the merges. Is there a particular reward item you wanted to add this functionality for?

Reply:
@zarillion I'm currently working on a Timewalking HandyNotes plugin that uses the same core from your current code. One of the rewards is limited to Warrior, Paladin, and Death Knight which is why I introduced the ability to add multiple classes for rewards.

I hadn't had a chance to ask if you a) are you interested in hosting the Timewalking plugin? I did have to make a minor change because there is no `ns.expansion`. and b) if you aren't interested in hosting the Timewalking plugin am I able to share it on my own using a duplicate of the core folder?

Reply:
Its hard to say without seeing the content the plugin adds and how it would fit into the current plugin-per-expansion setup we have in this main repository. You have permission to copy the `core/` directory and any future updates we make to it to support a separate Timewalking addon for now and we can always re-evaluate if it fits in this repository later.

Reply:
I just moved my [HandyNotes: Timewalking](https://github.com/wyldclaw/handynotes-timewalking) repo from Private to Public now that I was able to add `core/` to the plugin if you care to check it out sometime.